packages/next/src/index.ts

```diff
@@ -147,10 +147,10 @@ function getRealNextVersion(entryPath: string): string | false {
     // package.json. This allows the builder to be used with frameworks like Blitz that
     // bundle Next but where Next isn't in the project root's package.json
 
-    // NOTE: `eval('require')` is necessary to avoid bad transpilation to `__webpack_require__`
```

Comment:
The note here is still relevant, just needs to be updated to say esbuild, right?

Reply:
I'll put a single comment where `require_` gets created / exported, so that it shows up in vs code.